runtime/onert/frontend/circle/src/circle_loader.cc

```diff
@@ -72,6 +72,12 @@ class CircleLoader final : public base_loader::BaseLoader<LoaderDomain, CircleLo
 public:
   using BaseLoader::BaseLoader;
 
+  std::vector<BuiltinOperator> getOptionalInputOplist() override
```

Comment:
IMO, we can introduce `bool allowOptionalInput(BuiltinOperator)` instead of return vector to reduce return overhead.

Reply:
@hseok-oh Thanks, I think that's better, I applied.